src/rules/validator.ts

```diff
@@ -67,7 +69,7 @@ export const validateConfig = (config: ConfigurationFile): ConfigurationFile | n
       validatedConfig.rules[i] = validate<BasicRule>(rule, basicRuleSchema, { message });
     } else if (type === "debug") {
       validatedConfig.rules[i] = validate<DebugRule>(rule, ruleSchema, { message });
-    } else if (type === "and" || type === "or") {
+    } else if (type === "and" || type === "or" || type === "and-distinct") {
       validatedConfig.rules[i] = validate<AndRule>(rule, otherRulesSchema, { message });
```

Comment:
`validate<AndRule>` is this correct to have only AndRule for all types?

Reply:
Yes, because, aside from the Basic rule, every other rule have the exact same structure:

They all have:
```yaml
rules:
  - name: Example rule
    condition:
      include: 
        - '.*'
      exclude: 
        - 'example'
    type: and-distinct | and | or
    reviewers:
      - teams:
        - team-example
        min_approvals: 2
      - teams:
        - team-abc
```